src/index.ts

```diff
@@ -1,3 +1,6 @@
+import * as dotenv from 'dotenv';
+dotenv.config({ path: `${__dirname}/../.env` });
```

Comment:
```suggestion
dotenv.config({ path: `${process.cwd()}/.env` });
```

Essa função do node retorna a raiz do repo. Seria a pasta onde fica o `package.json`. Mais curto o caminho, né?

Reply:
Bem melhor... feito as alterações.